README.md

```diff
@@ -32,14 +32,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+Document Object Model. How you interact and manipulate items and content of a webpage. Using Javascript to manipulate HTML and CSS.
```

Comment:
It's more so *where* we interact with our site's original content. It's a replica of it converted into in nodes and objects: https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction